.github/SETUP_SECRETS.md

```
@@ -0,0 +1,170 @@
+
+---
+
+### 2. Code Coverage Secret (Optional)
+
+#### `CODECOV_TOKEN`
+
+**Purpose**: Upload test coverage reports to Codecov
+
+**How to get it**:
+
+1. Go to https://codecov.io
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)